### ISSUE TITLE
Crypto module in std.

### ISSUE THREAD

**Issue by vikasgola** [dlang/projects#58]
## Description
It would be great and helpful if a crypto module can be included in std.

## What are rough milestones of this project?
To include some basic crypto methods in the D language

## Recommended skills
- medium D language experience
- cryptography knowledge

## Point of Contact
@vikasgola

## References
https://forum.dlang.org/thread/qjsmnyldyajxkttezdib@forum.dlang.org

**Comment by andre2007** [comment]
Yes, I highly miss a RSA signature validation within Phobos. One use case for that is JWT. Maybe this could be used as inspiration https://www.libtom.net/.

Should we add the gsoc2021 label?

**Comment by vikasgola** [comment]
@andre2007 Yes, sure.
